src/extension.ts

```diff
@@ -426,7 +426,12 @@ export class ExtensionManager implements vscode.Disposable {
                 // We have an active kit. We're good.
                 return true;
             }
-            // No kit? Ask the user what they want.
+            // // No kit? Is enable kit scan on?
+            if (!this.workspaceConfig.enableKitScan) {
+                await cmakeProject.kitsController.setKitByName("");
```

Comment:
If it's possible, it might be more clear to explicitly set this to `SpecialKits.Unspecified` here.